doc/user/static/environmentd-openapi.yml

```diff
@@ -0,0 +1,159 @@
+openapi: 3.0.3
+info:
+  title: environmentd
+  version: 0.1.0
+paths:
+  /api/sql:
+    post:
+      summary: execute sql statement(s)
+      parameters:
```

Comment:
Should we document this as an array instead, and list things like `cluster` in a doc string?